src/main/java/com/example/onlinebookstore/dto/OrderItemResponseDto.java

```diff
@@ -0,0 +1,16 @@
+package com.example.onlinebookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
```suggestion
@Data
```
Same for all other request/response DTO's

Reply:
done